### ISSUE TITLE
[Question] ops.Normalize for volumetric data

### ISSUE THREAD

**Issue by JuanFMontesinos** [NVIDIA/DALI#2209]
Hi,
I was wondering how does `ops.Normalize` behave with volumetric data (videos) when `batch=False`
Specifically I was wondering if it normalized each "temporal" frame independently or all together.

Would it match the following code for no scale and no shift?

EDIT:
BTW is it possible to use ops functionals without the nvidia pipeline?
It's useful to have simple implementation for inference cases and reproducibility.

```
class VideoNormalization:
    """
    Mean-std Normalization for volumetric data of shape *xC along the dimension c.
    Typical data can be TxHxWxC (volumetric data) or HwWxC
    """

    def __init__(self, epsilon: float = 0.):
        self.epsilon = epsilon

    def __call__(self, array: Union[torch.tensor, np.array]):
        if isinstance(array, torch.Tensor):
            kw = {'dim': 0}
        elif isinstance(array, np.ndarray):
            kw = {'axis': 0}
        else:
            raise TypeError(f'VideoNormalization only accepts torch.Tensor or np.ndarray as input')
        C = array.shape[-1]
        array_ = array.reshape(-1,C)
        mean = array_.mean(**kw)
        std = array_.std(**kw)
        return (array - mean) / (std + self.epsilon)

    def _torch(self):
        tensor = torch.rand(7, 5, 5, 3)
        return tensor, self(tensor)

    def _numpy(self):
        array = np.random.rand(7, 5, 5, 3)
        return array, self(array)
```

**Comment by JanuszL** [comment]
Hi,
You can use `axes` to indicate dimensions along which the input is normalized. `Temporal` is one of these dimensions.
> BTW is it possible to use ops functionals without the nvidia pipeline?
You cannot use that directly but you can create simple code that would mimic that, something like:
```  
pipe = Pipeline(batch_size=batch_size, num_threads=4, device_id=0, exec_pipelined=False, exec_async=False)
data = fn.external_source(source = [your_data], no_copy=True)
processed = fn.normalize(data, batch  = True)
pipe.set_outputs(processed)
pipe.build()
out = pip.run()
```
You can pass your input data, numpy array, cupy array or PyTorch tensor and providing `no_copy` it will work directly on the provided memory (remember to assign the right device to the external source that corresponds to the place where data is to make zero-copy working). The output from DALI support CUDA array interface, however, I don't think PyTorch can consume it directly, so probably you need to go through cupy:
```
from torch.utils.dlpack import from_dlpack
import cupy as cp
capsule = cp.asanyarray(dali_out).toDlpack()
torch_tensor = from_dlpack(capsule )
```

**Comment by JuanFMontesinos** [comment]
Thanks a lot.
